includes/wikia/helpers/WikiaIFrameTagBuilderHelper.class.php

```diff
@@ -1,10 +1,6 @@
 <?php
 class WikiaIFrameTagBuilderHelper extends WikiaTagBuilderHelper {
 
-	public function isMobileSkin( ) {
```

Comment:
I moved it to `WikiaTagBuilderHelper` because it has nothing to do with iframes.